thorlcr/graph/thgraphslave.cpp

```diff
@@ -1270,6 +1270,9 @@ bool CSlaveGraph::serializeStats(MemoryBuffer &mb)
     stats.setStatistic(StSizePeakMemory, processActiveInfo.getPeakResidentMemory());
     jobS->querySharedAllocator()->queryRowManager()->reportSummaryStatistics(stats);
 
+    IGraphTempHandler *tempHandler = owner ? queryTempHandler(false) : queryJob().queryTempHandler();
+    if (tempHandler)
+        stats.setStatistic(StPeakSizeGraphSpill, tempHandler->getActiveUsageSize());
```

Comment:
I think this is going to overwrite with the passed in value,
needs to b stats.mergeStatistic so it uses the correct StatsMergeAction I believe.